README.md

```diff
@@ -75,6 +75,30 @@ For our need to forcibly close connections to unelected masters and handle keepe
 
 We are open to alternative solutions (PRs are welcome) like using haproxy if they can met the above requirements. For example, an hypothetical haproxy based proxy needs a way to work with changing ip addresses, get the current cluster information and being able to forcibly close a connection when an haproxy backend is marked as failed (as a note, to achieve the latter, a possible solution that needs testing will be to use the [on-marked-down shutdown-sessions](https://cbonte.github.io/haproxy-dconv/configuration-1.6.html#5.2-on-marked-down) haproxy server option).
 
+### How does Stolon differ from ____?
+
+#### Pure Kubernetes
+
+Stolon does not require a shared volume be attached to whatever node the Postgres container is running on. **Is Stolon faster than K8's pod process?? Is that another reason? I can't imagine it would be that different**. With a pure Kubernetes approach and something like an EBS volume, you can have a single Postgres master and even slave nodes with replication that could serve as read only servers. In the event of a master failure, K8S can move the DB container around and repoint the service as necessary.
```

Comment:
I don't get your question... What does it mean "I'm guessing if you're using Stolon"?

> Stolon isn't really affected one way or the other by the (current) lack of fencing in k8s?

Lack of fencing in k8s depends on how you are setting up your pods. The different issues are stated in kubernetes/kubernetes#34160. So stolon is affected if you're encountering that issues like any other program running under k8s. 
For example in a scenario where you have N stolon-keeper pods manually defined with their PV (no petset) and X sentinels the lack of fencing will be an issue only if you're using iscsi or other form of pv that actually, due to the lack of fencing, can be mounted on two nodes causing data corruption. But we can't do anything to avoid this since the problem is in the underlying infrastructure (k8s) and needs to be fixed there. So you'll have to restore from backup or, with stolon you can detect this corruption's and just switch to an healthy standby.